api/OEP_API_tutorial_part2.ipynb

```diff
@@ -136,17 +137,19 @@
    "metadata": {},
    "source": [
     "## 2. Include filters\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "A filter option is applied at the end of the result URL like this \".../rows/<b>?</b>where=filter_statement\" and is processed as HTTP/get request by the API.<br>\n",
+    "</div>\n",
     "### 2.1 where"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
+    "# insert 'columnname=value'\n",
```

Comment:
Is it a comment, or commented code? If it is commented code it should not be pushed.

Reply:
This is an example of what values should be used there

Reply:
Ok :)